packages/jest-emotion/src/index.js

```diff
@@ -35,11 +37,50 @@ function getNodes(node, nodes = []) {
   return nodes
 }
 
+function copyProps(src, target) {
```

Comment:
you are defining properties on `src` - which means that those are slightly wrongly named, should be other way around. The first parameter is a target (we copy props onto it) and the second one is a source (we copy from it)